tool/cli.py

```diff
@@ -100,11 +100,52 @@ def list_branches():
     print(tabulate(a.list_branches(), tablefmt="fancy_grid"))
 
 
+@click.command(name="labels",
```

Comment:
this could be confusing to new users: could we split the command to two? `list-labels` and `sync` or `update` `-labels`